research-intro.md

```diff
@@ -0,0 +1,210 @@
+* Koshy et al. [An Analysis of Anonymity in Bitcoin Using P2P Network Traffic](https://www.ifca.ai/fc14/papers/fc14_submission_71.pdf)
+
+A complementary approach to attacking privacy involves transaction graph analysis.
+Such deanonymization methods and countermeasures have been described in:
+
+* Androulaki et al. [Evaluating User Privacy in Bitcoin](https://fc13.ifca.ai/proc/1-3.pdf)
+* Bonneau et al. [Mixcoin: Anonymity for Bitcoin with Accountable Mixes](https://eprint.iacr.org/2014/077)
+* Meiklejohn et al. [A Fistful of Bitcoins: Characterizing Payments Among Men with No Names](https://cseweb.ucsd.edu/~smeiklejohn/files/imc13.pdf)
+* Reid and Harrigan. [An Analysis of Anonymity in the Bitcoin System](https://arxiv.org/abs/1107.4524)
+* Ruffing et al. [CoinShuffle: Practical Decentralized Coin Mixing for Bitcoin](https://petsymposium.org/2014/papers/Ruffing.pdf)
+
+
+## Game Theory
+
+Bitcoin's core properties assume that network participants act in their best interest.
+Research challenges in the area include formalizing the actions of the network participants, identifying undesired deviation strategies, and coming up with ways to mitigate those.
+Notable papers that consider the game-theoretic properties of Bitcoin include:
+
+* Babaioff et al. [On Bitcoin and Red Balloons](https://arxiv.org/abs/1111.2626)
+* Carlsten et al. [On the Instability of Bitcoin Without the Block Reward](https://dl.acm.org/doi/10.1145/2976749.2978408)
+* Courtois and Bahack. [On Subversive Miner Strategies and Block Withholding Attack in Bitcoin Digital Currency](https://arxiv.org/abs/1402.1718)
+* Eyal and Sirer. [Majority is not Enough: Bitcoin Mining is Vulnerable](https://arxiv.org/abs/1311.0243) - the paper introduces _selfish mining_ attack where a malicious miner gets more than its fair share of block subsidy by strategically withholding or revealing blocks they mine.
+* Garay et al. [The Bitcoin Backbone Protocol: Analysis and Applications](https://eprint.iacr.org/2014/765)
+* Kroll et al. [The Economics of Bitcoin Mining, or Bitcoin in the Presence of Adversaries](https://asset-pdf.scinapse.io/prod/2188530018/2188530018.pdf)
+
+
+## Economics
+
+Bitcoin can also be viewed from the economics perspective.
+Papers on economic analysis of Bitcoin include:
+
+* Budish. [The Economic Limits of Bitcoin and Anonymous, Decentralized Trust on the Blockchain](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=4148014)
+* Böhme et al. [Bitcoin: Economics, Technology, and Governance](https://www.aeaweb.org/articles?id=10.1257/jep.29.2.213)
+* Huberman et al. [Monopoly without a Monopolist: An Economic Analysis of the Bitcoin Payment System](https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3025604)
+* Kayal and Rohilla. [Bitcoin in the economics and finance literature: a survey](https://link.springer.com/article/10.1007/s43546-021-00090-5)
+
+
+
+# Research Areas in Lightning
```

Comment:
I thought about this, but was hesitant to include it because a) it's not an easy read; b) it doesn't reflect the current state of the protocol. From personal experience, I remember spending quite some effort to read it carefully, only to  discover later on that the actual specs don't have "breach remedy transactions" in the same shape as the paper introduces.

Perhaps it should be included for historical reasons (like the Nakamoto whitepaper) with a comment that the protocol has diverged from it considerably?

Reply:
Yes, seems like a good idea to treat these two papers similarly.